packages/styled-components/src/models/InlineStyle.ts

```diff
@@ -1,4 +1,5 @@
 import transformDeclPairs from 'css-to-react-native';
+import { DynamicColorIOS, PlatformColor } from 'react-native';
```

Comment:
Does this require a major version bump? Not sure how long these exports have been around for